remoulade/api/main.py

```diff
@@ -78,6 +82,28 @@ def requeue_message(message_id):
         return {"error": "requeue message in a pipeline not supported"}, 400
 
 
+@app.route("/messages/result/<message_id>")
+def get_results(message_id):
+    from ..message import get_encoder
+
+    max_size = 1e4
+    try:
+        result = Result(message_id=message_id).get()
+        if isinstance(result, BackendResult):
```

Comment:
result is not a BackendResult, take a look at .get() mehtod. you can remove both next lines